pkg/apis/pipeline/v1/param_types.go

```diff
@@ -116,8 +116,7 @@ type Param struct {
 	Value ParamValue `json:"value"`
 }
 
-// extractParamValuesFromParams get all param values from params
-func (ps Params) extractParamValuesFromParams() []string {
+func (ps Params) extractParamValues() []string {
```

Comment:
can you please update the docstring instead of removing it?

Reply:
"this comment is unnecessary imo" - @Jerop
@lbernick I thought Jerop was suggesting removing it altogether..Happy to do either.

Reply:
sgtm if it's updated to something useful, not just saying what the name already says

Reply:
okay I agree that the name is pretty straightforward. 

I can add doc string that says 
`// extractParamValues extracts all the values in a parameter`

or leave it without docstring

Thoughts? @lbernick 
cc: @jerop

Reply:
I vote that we don't need that docstring, the name is clear